Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -1163,7 +1164,9 @@ - (void)setBarButtonItemsForDisappearingMessagesConfiguration:
             + kTitleVSpacing);
     self.navigationBarTitleLabel.frame
         = CGRectMake(0, 0, titleViewWidth, self.navigationBarTitleLabel.frame.size.height);
-    self.navigationBarSubtitleLabel.frame = CGRectMake(0,
+    self.navigationBarSubtitleLabel.frame = CGRectMake((self.view.isRTL ? self.navigationBarTitleView.frame.size.width
```

Comment:
Yes it would but iOS auto layout doesn't work properly inside custom bar button items.